CMakeLists.txt

```diff
@@ -40,8 +40,10 @@ if (NOT DXC_PATH)
     if (DEFINED ENV{VULKAN_SDK})
         if (CMAKE_HOST_WIN32)
             set(DXC_PATH "$ENV{VULKAN_SDK}\\Bin\\dxc.exe")
+            set(GLSLC_PATH "$ENV{VULKAN_SDK}\\Bin\\glslc.exe")
```

Comment:
You might want to separate this from the DXC logic, this will only trigger if (NOT DXC_PATH) and (DEFINED ENV{VULKAN_SDK})